src/components/LeftPanel/LeftPanel.js

```diff
@@ -61,7 +82,7 @@ class LeftPanel extends React.Component {
           <LeftPanelTabs />
         </div>
 
-        <ResizeBar />
+        <ResizeBar onWidthChange={this.onWidthChange} style={innerStyle}/>
```

Comment:
Use `right: 0` for the ResizeBar so we don't need to pass a style prop down to it

Reply:
I updated it to use `right: 0`